packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -52,8 +52,7 @@ class AndroidView extends StatefulWidget {
   ///
   /// The `viewType`, `hitTestBehavior`, and `gestureRecognizers` parameters must not be null.
   /// If `creationParams` is not null then `creationParamsCodec` must not be null.
-  AndroidView({ // ignore: prefer_const_constructors_in_immutables
-                // TODO(aam): Remove lint ignore above once dartbug.com/34297 is fixed
+  const AndroidView({
```

Comment:
I don't know @amirh 's full plans for this, but maybe we shouldn't change this to `const` in case he needs to add behavior to the constructor.

Reply:
He's OOO for another week I believe

Reply:
Thanks for the info.

Reply:
ping @amirh

Reply:
Sorry for the delayed response!
Should be safe to make this const, thanks!